python/llm/src/bigdl/llm/ggml/convert_model.py

```diff
@@ -16,47 +16,99 @@
 import os
 import time
 from pathlib import Path
-
 from bigdl.llm.ggml.convert import _convert_to_ggml
 from bigdl.llm.ggml.quantize import quantize
+from bigdl.llm.utils.common import invalidInputError
+import argparse
+import tempfile
 
 
 def convert_model(input_path: str,
-                  output_path: str,
+                  output_dir: str,
```

Comment:
should we just call it `output_path`, which is similar to `input_path` and we don't need to change other code.

Reply:
done